fastlane/Fastfile

```diff
@@ -1,16 +1,9 @@
-require 'yaml'
-
-#
-# `options` contains several parameter:
-#
-# version_code: The new version code
-# version_name: The new version name
-# package_name: The name of the app in Sentry
-# sourcemap_path: The path to the sourcemap file to upload
```

Comment:
Why did you remove this docu? I think we need a way to document options or check for them in the line.

Reply:
Because it was pretty inconsistent. And if you look at the docu above the function, you may as well look in the function itself to see what options are necessary.

Reply:
I'll readd it